### PR TITLE
Feature/maw1 32 navigation with framework

### PR DESCRIPTION
**Pull request by EliottJaquierEdu**

### CODE REVIEW

File: views/exercises/answering/fulfillment.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+include $_SERVER['DOCUMENT_ROOT'] . "/../views/headers/heading-answering.php" ?>
```
Comment (drutz-cpnv) [inline views/exercises/answering/fulfillment.php:2]: You MUST NOT use `require` nor `include` instead use View concatenate

File: views/exercises/answering/fulfillment.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<main class="container">
+    <h1>Your take</h1>
+    <p>If you'd like to come back later to finish, simply submit it with blanks</p>
+    <form action="/exercises/7/fulfillments" accept-charset="UTF-8" method="post">
```
Comment (drutz-cpnv) [inline views/exercises/answering/fulfillment.php:7]: You can already use `url` method in your views to generate your urls

File: views/templates/base.php [hunk 3]
```diff
@@ -6,9 +6,10 @@
     <title>Exercise Looper Clone</title>
     <script src="https://kit.fontawesome.com/f5c1a22e81.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/milligram/1.4.1/milligram.min.css">
-    <link rel="stylesheet" href="build/app.css">
+    <link rel="stylesheet" href="/build/app.css">
+    <script src="/assets/rails-ujs.js"></script>
 </head>
 <body>
-
+<?= $content ?>
```
Comment (drutz-cpnv) [inline views/templates/base.php:13]: You will have to change `$content` to `$this->body`

**Review by drutz-cpnv — CHANGES_REQUESTED** [review]

**Comment by EliottJaquierEdu** [comment]
All resolved, feel free to re-review or merge!

**Pull request merged**